org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/BasicScanTest.java

```diff
@@ -226,8 +225,8 @@ private void checkMetadataScannables(final ScanModel scanModel, NXinstrument ins
 		}
 	}
 
-	private IRunnableDevice<ScanModel> createStepScan(IScannable<?> monitor,
-			                                          IScannable<?> metadataScannable,
+	private IRunnableDevice<ScanModel> createStepScan(IScannable<?> monitorPerPoint,
+			                                          IScannable<?> monitorPerScan,
 			                                          int... size) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)